### DISCUSSION TITLE
山上的风车 | /posts/windmill-on-the-hill/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#651] [Announcements]
https://yuanfan.rbind.io/posts/windmill-on-the-hill/

**Comment by hongtaoh** [comment]
你这个 slug 感觉不错哦，windmill on the hill 很有诗意。我在翻看你过往博客的英文网址，感觉你英语很棒啊。

**Reply by earfanfan** [reply]
不，郝糊涂你又犯糊涂了，我的英文很烂的说，而且没有任何语言方面的才能。windmill on the hill，这个就是山上的风车的直译，只是刚好押韵了。等等，其实我不知道英文是否也有押韵一说，原本你感觉到的诗意可以给我解释一下是撒吗？

**Reply by hongtaoh** [reply]
我一开始倒没注意到押韵，只是觉得这个意境很好，另外 windmill 这个也是一个不太常见的词。还有就是 on the hill 这个搭配其实已经看出你的英语水平了。判断一个人的英语水平，比较直接的是看副词和介词用得怎么样。

**Reply by earfanfan** [reply]
额，虽然如果我说咱俩是“瞎猫碰上死耗子”这样同时贬损了我俩，但其实你用来判断的方法应该是对的，只是我真的是“死耗子”。去年我追了一部电视剧《新闻女王》，里面有一首《Crystal clear》，歌词戳[这里](https://lyrics.net.cn/lyrics/71899)，第一句就是“Feel the earth on the hill”。windmill 不常见，是因为这个词语对应的意象在生活中也确实不常见呀。

我忽然感觉到自己应该深刻地反省一下，其实我心里是能感受到每个人类的美好之处的，但我出声夸赞的时候却很少，尤其是去你的博客溜达的时候，吐槽太多了。那回头我也调整一下，多去你那里拍拍你的马屁试试。

**Reply by earfanfan** [reply]
又及，那部电视剧里也有一个角色提到了你最近写到博客里面的一句话，你写的我记得是“stay hungry, stay foolish”，而她是“stay hungry, stay stupid”。

**Reply by hongtaoh** [reply]
袁姐好有文化。
